### PR TITLE
Content updates (Auth landing, Update institution profile)

### LINKED ISSUE #353: [Pages] Content updates
- [x] Authenticated landing
- [x] Update institution profile

### PR DESCRIPTION
**Pull request by meissadia**
Closes #353 

## Changes

- Update institution profile
  - Update intro paragraph content
  - Update RSSD label
  - Reorder fields
- Authenticated landing - Review your financial institution profile
  - Update intro paragraph content

## How to test this PR

1. Visit http://localhost:8899/landing
2. Verify content
3. Visit http://localhost:8899/institution/123456789TESTBANK123/update (or any institution you are associated with)
2. Verify content

### Auth. Landing page
screencapture-localhost-8899-landing-2024-04-01-16_42_34

### Update institution profile
screencapture-localhost-8899-institution-123456789TESTBANK123-update-2024-04-01-16_42_26

### CODE REVIEW

File: src/pages/Summary/Summary.data.tsx [hunk 1]
```diff
@@ -120,9 +120,13 @@ function ChildrenSuccessInstitutionProfileUpdate(): JSX.Element {
     <>
       Please allow 24-48 hours for a response during normal business hours. If
       you need further assistance please{' '}
-      <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update your financial institution profile: Questions after submitting form'>
+      <Link
+        className='border-b-[1px]'
+        href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update your financial institution profile: Questions after submitting form'
```
Comment (billhimmelsbach) [inline src/pages/Summary/Summary.data.tsx:125]: Great! Do we need this for this other `Link` on the `Summary` page as well?

https://github.com/cfpb/sbl-frontend/pull/354/commits/fab5a725fc4704aaa09696eb9f46e1e646d1d3e8#diff-245c110e80bc6d42c2292f783a58eab95844704c4b0457ec4592f860d64570a0R110

File: src/pages/Filing/FilingHome.tsx [hunk 2]
```diff
@@ -23,8 +23,8 @@ function Home(): ReactElement {
     <div id='filing-home'>
       <Layout.Main id='main' layout='2-1' bleedbar classes='main-layout'>
         <Hero
-          heading='Get started filing your small business lending data'
-          subheading='Covered financial institutions are required to compile, maintain, and report information about small business lending.'
+          heading='Get started filing your lending data'
+          subheading='Covered financial institutions and voluntary reporters to maintain, report, and publicly disclose information about lending.'
```
Comment (billhimmelsbach) [inline src/pages/Filing/FilingHome.tsx:27]: I see you updated based on the figma, but I don't think it's right. Don't we still need `are required` here in this subheading, the sentence sounds weird without it? Also, I don't think covered financial institutions are required to publicly disclose information: that's what we do. They just need to report it.
Screenshot 2024-04-02 at 1 46 26 PM

**Review by billhimmelsbach — CHANGES_REQUESTED** [review]

File: src/pages/Filing/FilingHome.tsx (hunk 2 above)
Comment (billhimmelsbach) [reply src/pages/Filing/FilingHome.tsx:27]: I see on our website we say something about publicly disclosing stuff, but I think the "rule" subject there is doing some heavy lifting. If include the "are required" then I'm fine keeping the rest the same. 👍

Screenshot 2024-04-02 at 1 53 32 PM

Comment (meissadia) [reply src/pages/Filing/FilingHome.tsx]: Updated hero content per the design file. 

`Covered financial institutions are required to maintain, report, and publicly disclose information about lending to the CFPB.`

File: src/pages/Summary/Summary.data.tsx (hunk 1 above)
Comment (meissadia) [reply src/pages/Summary/Summary.data.tsx:130]: @billhimmelsbach Thanks for catching that. Updated.

**Review by billhimmelsbach — CHANGES_REQUESTED** [review]
Heyo! I think the cypress test is still failing?

**Pull request merged**